packages/@aws-cdk/aws-apprunner-alpha/lib/auto-scaling-configuration.ts

```diff
@@ -0,0 +1,181 @@
+import * as cdk from 'aws-cdk-lib/core';
+import { Construct } from 'constructs';
+import { CfnAutoScalingConfiguration } from 'aws-cdk-lib/aws-apprunner';
+
+/**
+ * Properties of the App Runner Auto Scaling Configuration.
+ */
+export interface AutoScalingConfigurationProps {
```

Comment:
Shouldn't we allow to specify [`tags`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-apprunner-autoscalingconfiguration.html#cfn-apprunner-autoscalingconfiguration-tags) as well?

Reply:
Looking at the other L2 Constructs, it seemed like there were few resources adding tags as properties, so I didn't add them.
Since tags can also be added through aspects, I thought they might be unnecessary like other resources. What do you think?
I would appreciate your opinion on the criteria for adding tags, as I'm not sure about it.

Reply:
Makes sense 👍 Thanks for clarifying!